lib/lotus.rb

```diff
@@ -9,4 +10,18 @@
 #
 # @see http://lotusrb.org
 module Lotus
+
+  # @since x.x.x
+  #
+  # @example Lotus.env => "development"
+  def self.env
+    Environment.new.environment
+  end
+
+  # @since x.x.x
```

Comment:
What do you think about adding description for this method? e.g.: `Check to see if specified environment matches the current environment`